pkg/utils/sentry/interceptors.go

```diff
@@ -35,6 +37,12 @@ func UnaryInterceptor(conf Config) grpc.UnaryServerInterceptor {
 		defer sentryRecovery(hub, ctx, repanic)
 		rep, err := handler(ctx, req)
 		if reportErrors && err != nil {
+			if level := errorLevel(err); level != sentry.LevelError {
+				hub.ConfigureScope(func(scope *sentry.Scope) {
+					scope.SetLevel(level)
+				})
+			}
+
 			hub.CaptureException(err)
```

Comment:
We were basically just capturing all RPC errors as Sentry exceptions, so now at least we have some level gradients. However, in production I also set `ENSIGN_SENTRY_REPORT_ERRORS="false"` since we have code to log errors to sentry in the code itself.